ramos/pool.py

```diff
@@ -35,11 +35,11 @@ def all(cls, *args, **kwargs):
 
         return [
             backend_class.create(*args, **kwargs)
-            for backend_class in cls._get_backends_classes()
+            for backend_class in cls.all_lazy()
         ]
 
     @classmethod
-    def _get_backends_classes(cls):
+    def all_lazy(cls):
```

Comment:
What about naming it `iterator()`? [Like that famous framework](https://docs.djangoproject.com/en/2.2/ref/models/querysets/#iterator)